widgetastic_manageiq/__init__.py

```diff
@@ -4725,3 +4727,12 @@ def select_plan(self, plan_name):
         """Find item by text and click to view details."""
         el = self._get_card_element(plan_name)
         self.browser.click(self.TITLE_LOCATOR, parent=el)
+
+    def get_progress_percent(self, plan_name):
+        """Returns datastore and vm progress bar percentage"""
+        el = self._get_card_element(plan_name)
+        _desc = [desc.text.lower()
```

Comment:
This is likely what's failing the `Black` linting in travis.

Reply:
Good news, black passed this time.